include/tarxx.h

```diff
@@ -599,36 +598,9 @@ namespace tarxx {
 
         // delete the copy constructor and copy assignment, as multiple instances to the same file are not supported
         tarfile(const tarfile&) = delete;
+        tarfile(tarfile&&) = delete;
         tarfile& operator=(const tarfile& other) = delete;
-
-        tarfile(tarfile&& other) noexcept
-        {
-            *this = std::move(other);
-        }
-
-        tarfile& operator=(tarfile&& other) noexcept
-        {
-            if (this == &other) {
-                return *this;
-            }
-
-            close();
-
-            type_ = other.type_;
-            mode_ = other.mode_;
-
-            file_name_ = std::move(other.file_name_);
-            file_ = std::move(other.file_);
-            callback_ = std::move(other.callback_);
-            stream_file_header_pos_ = other.stream_file_header_pos_;
-            stream_block_ = other.stream_block_;
-            stream_block_used_ = other.stream_block_used_;
-
-            platform_ = std::move(other.platform_);
-            stored_inos_ = std::move(stored_inos_);
-            other.close();
-            return *this;
-        }
+        tarfile& operator=(const tarfile&& other) = delete;
```

Comment:
Nevermind my previous comment about `tarfile`'s move-assignment being semantically bad. You deleted it here. Good!
Of course it would be great if you'd transplanted this into the earlier commit, but I'll not insist :)

Reply:
If you don't insist I won't transplant it :+1: